src/meta/proto-conv/src/schema_from_to_protobuf_impl.rs

```diff
@@ -223,6 +223,7 @@ impl FromToProto for ex::TableDataType {
         let x = match self {
             TableDataType::Null => new_pb_dt24(Dt24::NullT(pb::Empty {})),
             TableDataType::EmptyArray => new_pb_dt24(Dt24::EmptyArrayT(pb::Empty {})),
+            TableDataType::EmptyMap => todo!(),
```

Comment:
Will this `todo!()` be removed?

Reply:
Yes, and if proto-conv is modified, the a new proto version should be added, and the corresponding test should be added:

https://github.com/datafuselabs/databend/blob/c3bcdd27b631107df0d1e464b5d384fede8a3d53/src/meta/proto-conv/src/util.rs#L89-L94